package.json

```diff
@@ -5,7 +5,7 @@
     "compile": "tsc --build tsconfig.build.json",
     "compile:clean": "tsc --build tsconfig.build.json --clean",
     "watch": "tsc --build tsconfig.build.json --watch",
-    "release": "npm run clean && npm install && lerna publish --exact --force-publish=unmock-core --include-merged-tags",
+    "release": "npm run clean && npm install && lerna publish from-package --exact --force-publish=unmock-core --include-merged-tags",
```

Comment:
Not related to this PR, but why does it have `--force-publish=unmock-core`?

Reply:
No idea. Maybe @mikesol could shed some light on that :)
Maybe it was a safeguard or something.